modules/util/portal-tools-rest-builder/src/main/resources/com/liferay/portal/tools/rest/builder/dependencies/base_resource_impl.ftl

```diff
@@ -70,6 +75,33 @@ import com.liferay.portal.vulcan.util.ActionUtil;
 import com.liferay.portal.vulcan.util.LocalDateTimeUtil;
 import com.liferay.portal.vulcan.util.UriInfoUtil;
 
+<#if freeMarkerTool.isUseJavax(configYAML)>
```

Comment:
Instead of these big if elses, would it be posible to do something like:

```
import ${javaxPackage}.servlet.ServletContext;
```

This way we would not duplicate the lines and we just have to calculate the package prefix once.